sites/public/src/pages/listing/ext/[id].tsx

```diff
@@ -86,19 +87,24 @@ export async function getServerSideProps(context: {
   locale: string
 }) {
   let response: AxiosResponse
+  let jurisdiction: Jurisdiction
   try {
     const extUrl = `${process.env.BLOOM_API_BASE}/listings/${context.params.id}`
     response = await axios.get(extUrl, {
       headers: { language: context.locale },
     })
+    jurisdiction = await fetchJurisdictionByName(
+      process.env.BLOOM_API_BASE,
```

Comment:
Curious why this is `process.env.BLOOM_API_BASE` and the above is `runtimeConfig.getBackendApiBase()`?

Reply:
Forgot to clarify this! This is using the bloom api base since it's handling listings external to Doorway and the getBackendApiBase() returns the doorway proxy url or base url since its handling internal listings.